Doc/whatsnew/3.11.rst

```diff
@@ -1748,7 +1748,7 @@ Modules
   warnings have now been updated to note they will be removed in Python 3.12.
   (Contributed by Hugo van Kemenade in :issue:`47022`.)
 
-* The :mod:`lib2to3` package and :ref:`2to3 <2to3-reference>` tool
+* The :mod:`lib2to3` package and ``2to3`` tool
```

Comment:
```suggestion
* The ``lib2to3`` package and ``2to3`` tool
```

Reply:
I updated the PR to use ``!lib2to3`` syntax.

Reply:
Yes, using the ``` :mod:`!lib2to3` ``` syntax gives you the correct formatting. FTR, you could have used ``` :program:`2to3` ``` for correct formatting of the tool name.